### ISSUE TITLE
Dividing mask into regions.

### ISSUE THREAD

**Issue by cnu1439** [younggon2/Research-Segmentation-Lung-CXR-COVID19#1]
Hi, 

    Thanks for the pre-trained models. I have read the paper. while doing intensity normalisation the average intensity for out side lung regions is calculated for each region or for the entire scan ?. And how can I identify  LUR, RUR, LLR and RLR from final lung region mask ?.

Thanks,
Srinivas.

**Comment by younggon2** [comment]
Thanks for reading my paper and sending your question.

Please, refer to answers as below.

while doing intensity normalisation the average intensity for out side
lung regions is calculated for each region or for the entire scan ?.

-> The intensity for outside of lung regions were calculated for the
entire scans except for the lung regions segmented by the algorithms
and the background region. In here, the background region was
extracted with a certian threshold value 50 (range 0~255).

[image: image.png]

And how can I identify  LUR, RUR, LLR and RLR from final lung region mask ?.

-> To split the both lung regions (right and left) into upper and lower
region, carina structure was detected. The center point of the structure
was used as a reference point to split the upper and lower lung.
You can use the detection algorithm as well in my github like below.

https://github.com/younggon2/Research-Detection-CarinaHilar-CXR-COVID19

Thanks.

--------------------------------------------------------------
*Young-Gon Kim, PhD*

Research Professor
Transdisciplinary Department of Medicine & Advanced Technology
Seoul National University Hospital
101 Daehak-ro, Jongno-gu, Seoul 03080, Republic of KOREA
M: +82-10-5058-6988
T: +82-2-2072-4487
E: younggon2.kim@gmail.com



2020년 11월 6일 (금) 오후 4:36, cnu1439 <notifications@github.com>님이 작성:
